apiary.apib

```diff
@@ -2163,6 +2163,13 @@ Update an invoice.
                         + product_category_id: `e2314517-3cab-4aa9-8471-450e73449041` (string, optional)
         + invoice_date: `2016-02-04` (string, optional)
         + note: `Some comments about the invoice` (string, optional, nullable)
+        + discounts (array)
+            (object)
+                + value: 10 (number, required) - Values between 0 and 100
```

Comment:
This comment is specific to the "percentage" discount and won't make sense if there ever is a new type (which looks like it will come).

Shouldn't this comment be added to the "percentage type"?